src/app/components/team/Languages/AddLanguageAction.js

```diff
@@ -143,6 +143,7 @@ const AddLanguageAction = ({ team }) => {
                           description="Label to language selection dropdown"
                         />
                       }
+                      helpContent={<FormattedMessage id="addLanguageAction.selectLanguageHelp" defaultMessage="After adding this language, be sure to customize the chatbot’s responses in the tipline settings." description="Help text for next steps after selecting a language" />}
```

Comment:
This was help text that somehow I forgot to include in https://github.com/meedan/check-web/pull/1722